galgebra/mv.py

```diff
@@ -1615,12 +1615,8 @@ def Add(sdop1, sdop2):
         else:
             # convert values to multiplicative operators
             if isinstance(sdop1, Sdop):
-                if not isinstance(sdop2, Mv):
-                    sdop2 = sdop1.Ga.mv(sdop2)
                 sdop2 = Sdop([(sdop2, Pdop({}, ga=sdop1.Ga))], ga=sdop1.Ga)
             elif isinstance(sdop2, Sdop):
-                if not isinstance(sdop1, Mv):
-                    sdop1 = sdop2.Ga.mv(sdop1)
                 sdop1 = Sdop([(sdop1, Pdop({}, ga=sdop2.Ga))], ga=sdop2.Ga)
```

Comment:
Why 1 = 1 2 2

Reply:
`sdop1` is a scalar in this case, so it has no associated GA. We have to attache one to it, so we attach the ga from `sdop2`.

Really attaching a ga doesn't make sense here in the first place, but that's a wider-scoped change.

Reply:
Seeing these call sites makes one doubt the API design.

Reply:
Indeed, that's what started #200